user_docs/en/userGuide.t2t

```diff
@@ -2118,6 +2119,9 @@ The tone will increase in pitch every space, and for a tab, it will increase in
 - Both Speech and Tones: This option reads indentation using both of the above methods.
 -
 
+If you tick the "Ignore blank lines for line indentation reporting" checkbox, then indentation changes won't be reported for blank lines.
+This may be useful when reading a document where blank lines are used to separate indented bloks of text, such as in programming source code.

```

Comment:
Fix typo

```suggestion
This may be useful when reading a document where blank lines are used to separate indented blocks of text, such as in programming source code.
```